src/test/java/bio/terra/common/logging/LoggingTest.java

```diff
@@ -194,6 +194,10 @@ public void testStructuredLogging(CapturedOutput capturedOutput) {
 
   // Uses the JsonPath library to extract data from a given path within a JSON string.
   private <T> T readJson(String line, String path) {
+    if (line.isEmpty()) {
```

Comment:
Maybe mark the method `@Nullable` if this wasn't already understood.

Reply:
thanks, done.